### PR TITLE
Do not show lonely comma on empty region

### PR DESCRIPTION
**Pull request by amenk**
### Description
If the address is empty or partial, a lonley comma was shown

### CODE REVIEW

File: app/code/Magento/Checkout/view/frontend/web/template/shipping-information/address-renderer/default.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <!-- ko text: address().prefix --><!-- /ko --> <!-- ko text: address().firstname --><!-- /ko --> <!-- ko text: address().middlename --><!-- /ko -->
     <!-- ko text: address().lastname --><!-- /ko --> <!-- ko text: address().suffix --><!-- /ko --><br/>
     <!-- ko text: address().street --><!-- /ko --><br/>
-    <!-- ko text: address().city --><!-- /ko -->, <span data-bind="html: address().region"></span> <!-- ko text: address().postcode --><!-- /ko --><br/>
+    <!-- ko text: address().city --><!-- /ko --><!-- ko if: !(address().region.trim() == "") -->, <span data-bind="html: address().region"></span><!-- /ko --> <!-- ko text: address().postcode --><!-- /ko --><br/>
```
Comment (omiroshnichenko) [inline app/code/Magento/Checkout/view/frontend/web/template/shipping-information/address-renderer/default.html:11]: Use strict equality or check on string length.

**Comment by ishakhsuvarov** [comment]
Hi @amenk 
Could you please check and fix according to review comments? Let us know if you wish any assistance with this.
Thanks.

File: app/code/Magento/Checkout/view/frontend/web/template/shipping-information/address-renderer/default.html (hunk 1 above)
Comment (okorshenko) [reply app/code/Magento/Checkout/view/frontend/web/template/shipping-information/address-renderer/default.html:11]: we will fix this on our side. thank you for review

**Comment by okorshenko** [comment]
Hi @amenk 
Thank you for your contribution. Unfortunately, we can not accept this PR as is since it adds inconsistency to the rendering. 
We need to fix not only shipping information block, but address selection on the first step of checkout (when you are logged in and have few address in the address book), billing address rendering,  address book rendering on Customer Account page. 
So if we decided to change address rendering format, we must make it consistent. Otherwise, it will be worse than now

**Comment by amenk** [comment]
Yes feel free to fix all the the renderings,best by referring to a single
template.

**Comment by okorshenko** [comment]
Hi @amenk we will add this to our backlog. Closing this PR, since we need to deliver fixes to all places in one PR.

**Pull request closed without merging**